Data-Explorer/server.R

```diff
@@ -18,21 +14,21 @@
 
 ### Server ----
 
-credentials <- readRDS("data/admin/credentials.rds")
-
-function(input, output, session) {
-  if(credentials$authorisation=="Public"){
+#credentials <- readRDS("data/admin/credentials.rds")
 
-  } else if (credentials$authorisation=="Private"){
-    # Shinymanager Auth
-    res_auth <- secure_server(
-      check_credentials = check_credentials(credentials)
-    )
-    
-    output$auth_output <- renderPrint({
-      reactiveValuesToList(res_auth)
-    })
-  } 
+ function(input, output, session) {
```

Comment:
Should this not also be commented?

Reply:
No we need to leave that otherwise the app doesn't work

Reply:
Ok, didn't count the curly brackets properly sorry.